2025-06-23 14:02:15,679 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:06:38,192 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:28:51,157 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:28:51,974 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:29:06,580 WARNING: Login attempt failed: User admin not found [in C:\crime_mapping_visualization 0\app\routes\auth.py:247]
2025-06-23 14:29:18,430 WARNING: Login attempt failed: User admin not found [in C:\crime_mapping_visualization 0\app\routes\auth.py:247]
2025-06-23 14:32:28,012 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:32:28,908 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:32:54,780 INFO: Starting OTP generation process [in C:\crime_mapping_visualization 0\app\routes\auth.py:262]
2025-06-23 14:32:54,781 INFO: Generated OTP: 673306, Expiry: 1750660674 [in C:\crime_mapping_visualization 0\app\routes\auth.py:265]
2025-06-23 14:32:54,781 INFO: Attempting database connection for OTP storage [in C:\crime_mapping_visualization 0\app\routes\auth.py:267]
2025-06-23 14:32:54,783 INFO: Updating OTP for user_id: 2 [in C:\crime_mapping_visualization 0\app\routes\auth.py:271]
2025-06-23 14:32:54,784 INFO: Query: UPDATE user SET otp = %s, otp_expiry = %s WHERE user_id = %s, Values: ('673306', 1750660674, 2) [in C:\crime_mapping_visualization 0\app\routes\auth.py:274]
2025-06-23 14:32:54,786 INFO: OTP update query executed [in C:\crime_mapping_visualization 0\app\routes\auth.py:277]
2025-06-23 14:32:54,787 INFO: Database changes committed [in C:\crime_mapping_visualization 0\app\routes\auth.py:280]
2025-06-23 14:32:54,788 INFO: Database connection closed [in C:\crime_mapping_visualization 0\app\routes\auth.py:284]
2025-06-23 14:32:54,789 INFO: OTP generation successful for user admin [in C:\crime_mapping_visualization 0\app\routes\auth.py:287]
2025-06-23 14:32:56,866 INFO: Starting OTP generation process [in C:\crime_mapping_visualization 0\app\routes\auth.py:262]
2025-06-23 14:32:56,867 INFO: Generated OTP: 453177, Expiry: 1750660676 [in C:\crime_mapping_visualization 0\app\routes\auth.py:265]
2025-06-23 14:32:56,867 INFO: Attempting database connection for OTP storage [in C:\crime_mapping_visualization 0\app\routes\auth.py:267]
2025-06-23 14:32:56,869 INFO: Updating OTP for user_id: 2 [in C:\crime_mapping_visualization 0\app\routes\auth.py:271]
2025-06-23 14:32:56,870 INFO: Query: UPDATE user SET otp = %s, otp_expiry = %s WHERE user_id = %s, Values: ('453177', 1750660676, 2) [in C:\crime_mapping_visualization 0\app\routes\auth.py:274]
2025-06-23 14:32:56,871 INFO: OTP update query executed [in C:\crime_mapping_visualization 0\app\routes\auth.py:277]
2025-06-23 14:32:56,872 INFO: Database changes committed [in C:\crime_mapping_visualization 0\app\routes\auth.py:280]
2025-06-23 14:32:56,873 INFO: Database connection closed [in C:\crime_mapping_visualization 0\app\routes\auth.py:284]
2025-06-23 14:32:56,874 INFO: OTP generation successful for user admin [in C:\crime_mapping_visualization 0\app\routes\auth.py:287]
2025-06-23 14:33:00,948 INFO: OTP email sent to muyahooo1992@gmail.com [in C:\crime_mapping_visualization 0\app\routes\auth.py:297]
2025-06-23 14:33:03,636 INFO: OTP email sent to muyahooo1992@gmail.com [in C:\crime_mapping_visualization 0\app\routes\auth.py:297]
2025-06-23 14:40:53,098 INFO: Starting OTP generation process [in C:\crime_mapping_visualization 0\app\routes\auth.py:262]
2025-06-23 14:40:53,101 INFO: Generated OTP: 662746, Expiry: 1750661153 [in C:\crime_mapping_visualization 0\app\routes\auth.py:265]
2025-06-23 14:40:53,101 INFO: Attempting database connection for OTP storage [in C:\crime_mapping_visualization 0\app\routes\auth.py:267]
2025-06-23 14:40:53,104 INFO: Updating OTP for user_id: 3 [in C:\crime_mapping_visualization 0\app\routes\auth.py:271]
2025-06-23 14:40:53,105 INFO: Query: UPDATE user SET otp = %s, otp_expiry = %s WHERE user_id = %s, Values: ('662746', 1750661153, 3) [in C:\crime_mapping_visualization 0\app\routes\auth.py:274]
2025-06-23 14:40:53,110 INFO: OTP update query executed [in C:\crime_mapping_visualization 0\app\routes\auth.py:277]
2025-06-23 14:40:53,112 INFO: Database changes committed [in C:\crime_mapping_visualization 0\app\routes\auth.py:280]
2025-06-23 14:40:53,113 INFO: Database connection closed [in C:\crime_mapping_visualization 0\app\routes\auth.py:284]
2025-06-23 14:40:53,114 INFO: OTP generation successful for user im.nins [in C:\crime_mapping_visualization 0\app\routes\auth.py:287]
2025-06-23 14:40:57,853 INFO: OTP email sent to im.ninsdejesus@gmail.com [in C:\crime_mapping_visualization 0\app\routes\auth.py:297]
2025-06-23 14:43:37,969 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:43:45,945 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:46:06,075 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:46:25,216 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:47:55,755 INFO: Application startup [in C:\crime_mapping_visualization 0\app\__init__.py:25]
2025-06-23 14:48:21,531 INFO: Starting OTP generation process [in C:\crime_mapping_visualization 0\app\routes\auth.py:273]
2025-06-23 14:48:21,532 INFO: Generated OTP: 536361, Expiry: 1750661601 [in C:\crime_mapping_visualization 0\app\routes\auth.py:276]
2025-06-23 14:48:21,532 INFO: Attempting database connection for OTP storage [in C:\crime_mapping_visualization 0\app\routes\auth.py:278]
2025-06-23 14:48:21,534 INFO: Updating OTP for user_id: 3 [in C:\crime_mapping_visualization 0\app\routes\auth.py:282]
2025-06-23 14:48:21,535 INFO: Query: UPDATE user SET otp = %s, otp_expiry = %s WHERE user_id = %s, Values: ('536361', 1750661601, 3) [in C:\crime_mapping_visualization 0\app\routes\auth.py:285]
2025-06-23 14:48:21,547 INFO: OTP update query executed [in C:\crime_mapping_visualization 0\app\routes\auth.py:288]
2025-06-23 14:48:21,551 INFO: Database changes committed [in C:\crime_mapping_visualization 0\app\routes\auth.py:291]
2025-06-23 14:48:21,551 INFO: Database connection closed [in C:\crime_mapping_visualization 0\app\routes\auth.py:295]
2025-06-23 14:48:21,552 INFO: OTP generation successful for user im.nins [in C:\crime_mapping_visualization 0\app\routes\auth.py:298]
2025-06-23 14:48:26,366 INFO: OTP email sent to im.ninsdejesus@gmail.com [in C:\crime_mapping_visualization 0\app\routes\auth.py:308]
